Hmm, I don't really know what goes wrong here regarding tests in KDL. Build log shows several errors of this kind...

```
FAILED: [code=1] lib/kdl/test/CMakeFiles/kdl-test.dir/src/ranges/tst_chunk_by_view.cpp.o 
/run/ccache/bin/c++ -DCATCH_CONFIG_ENABLE_ALL_STRINGMAKERS=1 -I/run/build/trenchbroom/lib/kdl/src -O2 -pipe -g -Wp,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -O3 -DNDEBUG -std=c++20 -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -pedantic -Wconversion -Wshadow=local -MD -MT lib/kdl/test/CMakeFiles/kdl-test.dir/src/ranges/tst_chunk_by_view.cpp.o -MF lib/kdl/test/CMakeFiles/kdl-test.dir/src/ranges/tst_chunk_by_view.cpp.o.d -o lib/kdl/test/CMakeFiles/kdl-test.dir/src/ranges/tst_chunk_by_view.cpp.o -c /run/build/trenchbroom/lib/kdl/test/src/ranges/tst_chunk_by_view.cpp
In file included from /run/build/trenchbroom/lib/kdl/test/src/ranges/tst_chunk_by_view.cpp:24:
/run/build/trenchbroom/lib/kdl/src/kdl/ranges/chunk_view.h:189:45: error: ‘const struct kdl::ranges::chunk_view<V>::inner_iterator’ has no member named ‘parent_’ [-Wtemplate-body]
  189 |                                iter_swap(*x.parent_->current_, *y.parent_->current_)))
      |                                             ^~~~~~~
/run/build/trenchbroom/lib/kdl/src/kdl/ranges/chunk_view.h:189:67: error: ‘const struct kdl::ranges::chunk_view<V>::inner_iterator’ has no member named ‘parent_’ [-Wtemplate-body]
  189 |                                iter_swap(*x.parent_->current_, *y.parent_->current_)))
      |                                                                   ^~~~~~~
/run/build/trenchbroom/lib/kdl/src/kdl/ranges/chunk_view.h: In function ‘constexpr kdl::ranges::chunk_view<V>::outer_iterator::difference_type kdl::ranges::operator-(std::default_sentinel_t, const chunk_view<V>::outer_iterator&) requires  sized_sentinel_for<decltype(std::ranges::_Cpo::end((declval<_Tp&>)())), decltype(std::ranges::__access::__begin((declval<_Container&>)()))>’:
/run/build/trenchbroom/lib/kdl/src/kdl/ranges/chunk_view.h:99:66: error: ‘const struct kdl::ranges::chunk_view<V>::outer_iterator’ has no member named ‘parent’; did you mean ‘parent_’? [-Wtemplate-body]
   99 |       const auto dist = std::ranges::end(i->parent_->base_) - *i.parent->current_;
      |                                                                  ^~~~~~
      |                                                                  parent_
/run/build/trenchbroom/lib/kdl/src/kdl/ranges/chunk_view.h:102:67: error: ‘const struct kdl::ranges::chunk_view<V>::outer_iterator’ has no member named ‘parent’; did you mean ‘parent_’? [-Wtemplate-body]
  102 |                : detail::div_ceil(dist - i.parent_->remainder_, i.parent->n_) + 1;
      |                                                                   ^~~~~~
      |                                                                   parent_
```

I suspect compiler, Qt or something being too new. It's my understanding that Trenchbroom uses older Ubuntu and GCC13 to test stuff. This new runtime however is, if I've understood correctly, on top of GCC15. This is likely a runtime issue and the jump is just too big. I guess I try 6.9 then.

--- a/com.kristianduske.TrenchBroom.json
+++ b/com.kristianduske.TrenchBroom.json
@@ -1,7 +1,7 @@
 {
     "id": "com.kristianduske.TrenchBroom",
     "runtime": "org.kde.Platform",
-    "runtime-version": "6.10",
+    "runtime-version": "6.9",
     "sdk": "org.kde.Sdk",
     "command": "trenchbroom",
     "rename-icon": "trenchbroom",

